.github/ISSUE_TEMPLATE/UI_UX.md

```diff
@@ -0,0 +1,24 @@
+---
+name: UI/UX
+about: Suggest a UI/UX idea for this project
+title: In order to [achieve something] As a [role] I want [something to be added or
+  changed]
+labels: enhancement
```

Comment:
Maybe this should also add the [UI/UX label](https://github.com/coopcycle/coopcycle-web/issues?q=is%3Aopen+is%3Aissue+label%3AUI%2FUX)?

Reply:
Yes, that's what I was trying to accomplish, add it as a label :)

Reply:
It already exists 🙂

Reply:
But I mean I want to be able to add it here with a pink colour. So when people create an issue they can select UX/UI

![image](https://user-images.githubusercontent.com/51931067/105190121-9a945380-5b2d-11eb-9386-184d112987bc.png)